### PR TITLE
i18n(zh-cn): translate components.mdx

### PR DESCRIPTION
**Pull request by liruifengv**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- translate components.mdx

### CODE REVIEW

File: docs/src/content/docs/zh/guides/components.mdx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+	<TabItem label="Moons">木卫一、木卫二、木卫三</TabItem>
+</Tabs>
+```
+
+以上代码在页面上生成了以下选项卡：
+
+<Tabs>
+	<TabItem label="Stars">天狼星、织女星、参宿四</TabItem>
+	<TabItem label="Moons">木卫一、木卫二、木卫三</TabItem>
+</Tabs>
```
Comment (delucis) [inline docs/src/content/docs/zh/guides/components.mdx:78]: Can we translate “Stars” and “Moons” in these example tab `label`s?

File: docs/src/content/docs/zh/guides/components.mdx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+<Card title="看看这个">你想要突出显示的有趣内容。</Card>
+
+<CardGrid>
+	<Card title="Stars" icon="star">
+		天狼星、织女星、参宿四
+	</Card>
+	<Card title="Moons" icon="moon">
+		木卫一、木卫二、木卫三
+	</Card>
+</CardGrid>
```
Comment (delucis) [inline docs/src/content/docs/zh/guides/components.mdx:115]: Same here for the `title` in each `<Card>`.

File: docs/src/content/docs/zh/guides/components.mdx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+:::tip[提示]
+在主页上使用卡片网格来显示项目的关键特性。
+添加 `stagger` 属性来垂直移动第二列的卡片，并增加视觉效果：
+
+```astro
+<CardGrid stagger>
+	<!-- cards -->
```
Comment (delucis) [inline docs/src/content/docs/zh/guides/components.mdx:123]: This comment can probably be translated too. Maybe something like “Card components” would be better so you can still include a literal mention of “Card”?

**Review by delucis — COMMENTED** [review]
Thank you @liruifengv! Noticed a couple of details we can still translate.

**Comment by liruifengv** [comment]
> Thank you @liruifengv! Noticed a couple of details we can still translate.
> 

I've updated it.

**Review by delucis — APPROVED** [review]
Thanks @liruifengv!

**Pull request merged**